app/routes.py

```diff
@@ -1,2 +1,160 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+from app.models.helpers import validate_planet
 
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+# CREATE PLANET
+@planets_bp.route("", methods = ["POST"])
+def create_planet():
+	request_body = request.get_json()
+	
+	new_planet = Planet.create(request_body)
+
+	db.session.add(new_planet)
+	db.session.commit()
+
+	return jsonify(f"Planet {new_planet.name} has been successfully created"), 201 #use make response when you want to return something that is not json
```

Comment:
Looks great! We could also consider validating the request_body in a separate helper function.

Reply:
To add onto your comment about `make_response`, we can also use it when we want to add additional headers to a response. Here are some examples in the documentation:
https://flask.palletsprojects.com/en/2.0.x/api/#flask.make_response